template/addons/trpc/protected-router.ts

```diff
@@ -0,0 +1,17 @@
+import * as trpc from "@trpc/server";
+import { createRouter } from "./context";
+
+export function createProtectedRouter() {
```

Comment:
Throw a comment on top of this describing what it is if you can - ideally tsdoc style so it has good info on hover 🙏